src/posts/lucky_0.17.cr

```diff
@@ -0,0 +1,148 @@
+  title "Lucky 0.17 has been released!"
+  slug "lucky-0_17-release"
+
+  def published_at
+    Time.utc(year: 2019, month: 8, day: 16)
+  end
+
+  def summary
+    <<-TEXT
+    Lucky v0.17 is a huge release with many enhancements to Avram, Lucky's ORM,
+    as well as bug fixes and enhancements.
+    TEXT
+  end
+
+  def content
+    <<-MD
+    Lucky v0.17 is now out and has support for the newest Crystal (v0.30)!
+
+    Be sure to upgrade your version of Crystal, and take a look at our [UPGRADE NOTES](https://github.com/luckyframework/lucky/blob/master/UPGRADE_NOTES.md#upgrading-from-016-to-017) for help with migrating your app.
+
+    ### What's new?
+
+    We've made a ton of changes for this release, but we will highlight a few.
+    You can see a full list through our [CHANGELOG](https://github.com/luckyframework/lucky/blob/master/CHANGELOG.md#changes-in-v017).
+
+    ### Better primary key support
+
+    Before this release, your primary key fields would be auto-generated as `Int32` for you with an option to
+    use `UUID`. We've now added a new `primary_key` method wich lets you specify the type. Avram supports a
+    few different types now to include `Int16` and `Int64`.
+
+    ```crystal
+    def migrate
+      create :users do
+        # creates a primary key column named `id`
+        primary_key id : Int64
+      end
+    end
+    ```
+
+    Along with the support for new types, you can now also name your primary key whatever you need.
+    Generally you'd use the name `id` for your primary column, but there may be a case where you need to
+    customize that.
+
+    [Learn more about primary keys](#{Guides::Database::Migrations.path(anchor: Guides::Database::Migrations::ANCHOR_PRIMARY_KEYS)})
+
+    ### JSON and Array support
+
+    Postgres supports lots of different column types. Two really important ones used for complex structures
+    are the `jsonb` and array column types. Lucky 0.17 adds support into specify your `jsonb` fields as
+    `JSON::Any`, or use any of the basic postgres array fields.
+
+    ```crystal
+    table :users do
+      column preferences : JSON::Any
+      column tags : Array(String)
+    end
+    ```
+
+    Now we can use these columns with our model.
+
+    ```crystal
+    user = UserQuery.first
+
+    typeof(user.tags) #=> Array(String)
+    user.preferences["theme_color"].as_s #=> "dark_mode"
+    ```
+
+    [Learn more about column types](#{Guides::Database::Models.path(anchor: Guides::Database::Models::ANCHOR_COLUMN_TYPES)})
+
+    ### Multi-DB support
+
+    Avram now supports multiple databases at the same time. You can have some models that connect to your
+    primary database, and then a second set of models that connect to a legacy database if you wish.
+
+    There are a lot of different use cases for multiple database like:
+    * Secondary for special finance reporting
+    * Legacy DB support
+    * One for API, One for CMS
+    * Read / Write replica setup
+    * So many others...
```

Comment:
What do you think about adding a closing sentence instead of this extra item?

> Whatever your other databases are for, Avram is now better equipped to accommodate you.